sparse_ho/algo/forward.py

```diff
@@ -147,7 +147,7 @@ def get_beta_jac_iterdiff(
                 dual_gap = pobj[-1] - dobj
                 if verbose:
                     print("dual gap %.2e" % dual_gap)
-                assert dual_gap >= -100 * np.finfo('float').eps
+                assert dual_gap >= -100 * np.finfo('float').eps * dobj
```

Comment:
I changed because if dobj = 15000 as in the example, a dgap of -1e-12 is tolerable.    np.finfo('float') is the smallest x s.t 1 + x == x,  so it's relative to 1
@QB3